CIP-0120/README.md

```diff
@@ -0,0 +1,276 @@
+
+By formalizing the form of the constitution and its iterations, we aim to promote its longevity and accessibility.
+This is essential to ensure the effectiveness of the CIP-1694 governance model.
+
+This standard will impact how Ada holders read the Constitution but the main stakeholders for this are the tool makers who wish to read, render and write a constitution.
+
+### Safety
+
+Without describing best practices for the form and handling of the constitution, we risk the constitution document being stored in an insecure manner.
+By storing the constitution on a decentralized platform, we can ensure its immutability and permissionless access.
+This is a step to improve the longevity and accessibility of each constitution iteration.
+
+### Interoperability
+
+By defining a file extension and formatting rules for the constitution we ensure that tooling reading and writing the constitution will be interoperable.
+Furthermore we aim to make the role of constitution iteration comparison tools easier, by minimizing formatting and style changes between iterations.
+This will reduce compatibility issues between tools, promoting the accessibility of the constitution.
+
+### Usability
+
+Rich text formatting greatly enhances the readability of text, especially in large complex documents.
+Without the ability to format text, it could easily become cumbersome to read, negatively effecting the accessability of the Cardano constitution.
+
+## Specification
+
+The following specification SHOULD be applied to all constitution iterations.
+This standard could be augmented in the future via a separate CIP which aims to replace this one.
+
+### Terminology
+
+#### Capitalisation of key terms
+
+To avoid unnecessary edits, and therefore checksum changes, constitution authors MUST follow the following standard English capitalisation rules unless a translation language indicates otherwise:
+
+`Constitution` vs. `constitution`
+
+- The Constitution in effect — either the "initial" one or any updated version — is unique and therefore capitalised ("Constitution") as a proper noun.
+- Draft or proposed constitutions are not unique & therefore are not capitalised ("constitution") as a common noun.
+- "Cardano Constitution" is a very specific proper noun phrase (also a title) and so each word is capitalised.
+- The phrase "the Constitution", unless used non-specifically (e.g. "the constitution that voters prefer"), would generally be assumed to be _the_ Constitution and capitalised as a proper noun.
+
+`Ada` vs. `ada` vs. `ADA`
+
+- `Ada` is the currency, while `ada` indicates units of that currency (e.g. "Ada holders can accumulate more ada to increase their influence.")
+- `ADA` is the trading symbol (e.g. "Fluctuations in ADA might influence decisions about the Treasury.")
```

Comment:
might also clarify the use of ₳ ?
There is a glossary definition used for years having this definition:

**₳ ada**
When talking about the cryptocurrency, do not capitalize, unless at the beginning of a sentence. All caps should be used for the ticker symbol only. Plural is ada. Unicode U+20B3.

**ADA**
The ticker symbol for ada, like EUR or USD. If in doubt, prefer ada. On social media, use $ADA.

Reply:
@gufmar @Ryun1 I use `₳` all the time in my own notes, communications, spreadsheets, etc. but for Cardano constitution documents I believe the best clarification would be to tell people to avoid it... since it technically signifies an altogether different currency: https://en.wikipedia.org/wiki/Argentine_austral

Reply:
I would prefer to not allow this symbol in the constitution text due to its alternative meaning as raised by @rphair

Reply:
Also as per CIP meeting / @Ryun1 @Crypto2099 we are considering additional language to stick to the usual 8-bit character set (mainly to additionally restrict emojis) so @gufmar thanks for suggestion but I am resolving this.